src/NumericalAlgorithms/LinearOperators/Divergence.hpp

```diff
@@ -6,6 +6,7 @@
 
 #pragma once
 
+#include <boost/optional.hpp>
```

Comment:
why is this include added here?

Reply:
Derp, I decided I didn't like a previous design and this got left. Removed and squashed